bin/_test-helpers.sh

```diff
@@ -362,7 +362,14 @@ install_version() {
 
     (
         set -x
-        "$linkerd_path" install | kubectl --context="$context" apply -f - 2>&1
+        # TODO: Use a mix of helm override flags and CLI flags and remove this condition
+        # once stable-2.10 is out
+        edge_regex='(edge)-([0-9]+\.[0-9]+\.[0-9]+)'
+        if [[ "$version" =~ $edge_regex ]]; then
+          "$linkerd_path" install --set proxyInit.ignoreInboundPorts="1234\,5678" --set controllerLogLevel=debug | kubectl --context="$context" apply -f - 2>&1
```

Comment:
can you use a mix of helm and non-helm flags here?

Reply:
Makes sense. Updated.